test/Microsoft.AspNet.Mvc.FunctionalTests/MvcSampleTests.cs

```diff
@@ -100,7 +100,7 @@ public MvcSampleTests(MvcTestFixture<MvcSample.Web.Startup> fixture)
         {
             // Arrange
             var request = new HttpRequestMessage(HttpMethod.Get, "http://localhost/Home/ReturnUser");
-            request.Headers.Accept.Add(MediaTypeWithQualityHeaderValue.Parse("application/xml;charset=utf-8"));
+            request.Headers.Accept.Add(MediaTypeWithQualityHeaderValue.Parse("application/xml"));
```

Comment:
The functional tests don't use `Accept-Charset` much but unit tests such as [test/Microsoft.AspNet.Mvc.Core.Test/ObjectResultTests.cs](https://github.com/aspnet/Mvc/blob/e075a58ee8437300b560714ddae28dd0c45926ca/test/Microsoft.AspNet.Mvc.Core.Test/ObjectResultTests.cs) and [test/Microsoft.AspNet.Mvc.Formatters.Json.Test/JsonOutputFormatterTests.cs](https://github.com/aspnet/Mvc/blob/e075a58ee8437300b560714ddae28dd0c45926ca/test/Microsoft.AspNet.Mvc.Formatters.Json.Test/JsonOutputFormatterTests.cs) cover things fairly well.